tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -718,17 +730,22 @@ def test_positive_rhev_vm_power_on(self):
             ['Password', self.rhev_password, 'field'],
             ['Datacenter', self.rhev_datacenter, 'special select'],
         ]
-        name = gen_string('alpha')
+        cr_name = gen_string('alpha')
         with Session(self.browser) as session:
             make_resource(
                 session,
-                name=name,
+                name=cr_name,
                 provider_type=FOREMAN_PROVIDERS['rhev'],
                 parameter_list=parameter_list
             )
+            if self.compute_resource.power_on_status(
```

Comment:
we are using only one VM for these tests, and we do not know the state of VM when starting the relevant tests,
so in case of power on test, we need that the VM is switched off initially, in case of power off test, we need that the VM is switched on initially